IRremoteESP8266.cpp

```diff
@@ -702,23 +702,32 @@ static void ICACHE_RAM_ATTR read_timeout(void *arg __attribute__((unused))) {
 
 static void ICACHE_RAM_ATTR gpio_intr() {
   uint32_t gpio_status = GPIO_REG_READ(GPIO_STATUS_ADDRESS);
+  uint32_t now = system_get_time();
```

Comment:
but we already in iram. our declaration is ICACHE_RAM_ATTR gpio_intr(). still should call micros() ? need to check it won't change delays.